pkg/cmd/sync.go

```diff
@@ -89,7 +89,9 @@ var SyncCmd = &cobra.Command{
 
 		if len(symbol) > 0 {
 			if userConfig.Sync != nil && len(userConfig.Sync.Symbols) > 0 {
-				userConfig.Sync.Symbols = []string{symbol}
+				userConfig.Sync.Symbols = []bbgo.SyncSymbol{
+					{Symbol: symbol},
```

Comment:
what about sessionNames?

Reply:
session name is optional, when session name is empty, the symbol will be applied to all sessions.